packages/client/src/components/GameView/GameView.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from 'antd';
+import { PauseCircleOutlined, PlayCircleOutlined } from '@ant-design/icons/lib/icons';
```

Comment:
Насколько помню, ант предлагает иконки подключать так


```suggestion
import { PauseCircleOutlined, PlayCircleOutlined } from '@ant-design/icons';
```

через lib это что-то вроде костыля. Но я давно не пользовался антом, и мб всё изменилось, но раньше точно было так